packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -141,9 +180,11 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
     if (!files) {
       return;
     }
+    /* @conditional-compile-remove-from(stable): FILE_SHARING */
```

Comment:
There should be no need to conditionally compile this. `files` will always be empty.